mappings/net/minecraft/block/BlockDispenser.mapping

```diff
@@ -50,6 +50,8 @@ CLASS none/ama net/minecraft/block/BlockDispenser
 	METHOD a applyRotation (Lnone/asm;Lnone/apl;)Lnone/asm;
 		ARG 0 state
 		ARG 1 rotation
+	METHOD a getDispenseOffset (Lnone/co;)Lnone/de;
```

Comment:
`getDispenseLocation` then? `getDispenseSpawnPosition` is something else that comes to mind.